recipes/surfinBH/meta.yaml

```diff
@@ -0,0 +1,54 @@
+{% set name = "surfinBH" %}
+{% set version = "1.0.9" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: 9d3df12a256e06c2c3d72983ac8355f02184d0faf2d8c8ac7728585b8b8effc5
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install --no-deps --ignore-installed . -vv"
```

Comment:
You can drop the `--no-deps --ignore-installed`

Reply:
Thanks for the review, @mwcraig. I'd like to understand when this is/isn't necessary, but right now I'll remove it.

Reply:
conda-build sets up the environment such that these flags applied correctly without every recipe needing to explicitly do it. This wasn't always the case which is why older recipes have the flags included.